### PR TITLE
docs(ci): deployment documentation

### PR DESCRIPTION
**Pull request by ilikejames**
deploy-readme2

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -63,3 +63,38 @@ To read about aspects of Reactive Trader in greater depth, check out the followi
...
+The commit will be tagged with the latest semver version (master by default is a new minor version), the published
+docker images will be similarly tagged with the same version, and there will  be a version number published in
+the application's UI. The new version will be automatically deployed to the [`dev`](https://web-dev.adaptivecluster.com/) environment.
+
+![Master deployment](./docs/images/deploy-master.png "Master deployment")
+
+To get a version into another environment, tag the version with `env-<environment_name>` e.g. `env-demo`.
+
+```bash
+# Note: If there tag already exists, it will have to be deleted first before re-tagging
```
Comment (keithbloom) [inline README.md]: Will bad things happen if a new commit is tagged before the old one is removed?

Comment (rikoe) [inline README.md]: `If there tag` - should be `the` #pedant

File: README.md [hunk 2]
```diff
@@ -63,3 +63,38 @@ To read about aspects of Reactive Trader in greater depth, check out the followi
...
+## Deployment
+
+We practice _continous delivery. Every merge to master triggers a build and publishing of the application's images.
+The commit will be tagged with the latest semver version (master by default is a new minor version), the published
+docker images will be similarly tagged with the same version, and there will  be a version number published in
+the application's UI. The new version will be automatically deployed to the [`dev`](https://web-dev.adaptivecluster.com/) environment.
+
+![Master deployment](./docs/images/deploy-master.png "Master deployment")
+
+To get a version into another environment, tag the version with `env-<environment_name>` e.g. `env-demo`.
```
Comment (rikoe) [inline README.md]: Please include cmd-line instructions for doing this, and how to do with the GitHub website as well.

File: README.md (hunk 1 above)
Comment (ilikejames) [reply README.md]: No. You just wont be able to tag if it exists. :)

File: README.md (hunk 2 above)
Comment (ilikejames) [reply README.md]: Done :)

**Comment by ilikejames** [comment]
Happy to move this into its own DEPLOY.md document as it is getting quite long.

**Pull request merged**